### PR TITLE
Discuss choices of approximate models better in the manual.

### PR DESCRIPTION
**Pull request by bangerth**
Following up on a conversation we had this morning on the ASPECT call. How do people feel about some text of this kind?

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -1674,7 +1674,38 @@ \subsection{Approximate equations}
...
-\cite{KLKLZTTK10}.
+\cite{KLKLZTTK10}; a theoretical and practical comparison of many of
+these formulations using \aspect{} can be found in
+\cite{gassmoller2020formulations}.
+
+\note{Historically, the mantle convection community has typically used
+  one or another of these simplified models for computer simulations
+  -- oftentimes the simplest of them, the Boussinesq Approximation
+  (BA) discussed in Section~\ref{sec:Boussinesq}. These kinds of
+  approximations are often appropriate in many contexts; for example,
```
Comment (bobmyhill) [inline doc/manual/manual.tex:1685]:
```suggestion
  approximations are appropriate in many contexts; for example,
```

File: doc/manual/manual.tex [hunk 2]
```diff
@@ -2073,12 +2104,22 @@ \subsubsection{The isothermal/isentropic compression approximation (ICA)}
...
 incompressible mantle convection, we will now describe the different steps one has
-to take to use one of these approximations in a computation.
-This includes
+to take to use one of these approximations in a computation. As noted
+towards the beginning of Section~\ref{sec:approximate-equations}, one
+should choose a formulation that is adequate for the situation one
+wants to simulate, taking into account that other authors in previous
+studies may have used formulations not because they were suited to the
+situation but also because, possibly, they did not have software
+available that allows for more complete models.
```
Comment (bobmyhill) [inline doc/manual/manual.tex:2120]:
```suggestion
available that implemented the most suitable formulation.
```

File: doc/manual/manual.tex [hunk 3]
```diff
@@ -1674,7 +1674,38 @@ \subsection{Approximate equations}
...
+  situation better than equations
+  \eqref{eq:stokes-1}--\eqref{eq:temperature}, but \textit{because
+    simulation technology did not allow for anything else at the
+    time}.
+
+  This has changed today, and \aspect{} implements more realistic
+  models as discussed in this section and in
+  Section~\ref{sec:choosing-a-formulation}. As a consequence, you
+  should evaluate which formulation and approximation is
+  appropriate for what you want to do -- just because someone else in
```
Comment (anne-glerum) [inline doc/manual/manual.tex:1702]:
```suggestion
  appropriate for what you want to do. The fact that someone else in
```

File: doc/manual/manual.tex [hunk 4]
```diff
@@ -1674,7 +1674,38 @@ \subsection{Approximate equations}
...
+  it is worth pointing out that many older publications showing
+  mantle convection simulations did not
+  rely on these approximations because they describe the physical
+  situation better than equations
+  \eqref{eq:stokes-1}--\eqref{eq:temperature}, but \textit{because
+    simulation technology did not allow for anything else at the
+    time}.
+
+  This has changed today, and \aspect{} implements more realistic
+  models as discussed in this section and in
```
Comment (bobmyhill) [inline doc/manual/manual.tex:1699]:
```suggestion
  formulations as discussed in this section and in
```

File: doc/manual/manual.tex [hunk 5]
```diff
@@ -1674,7 +1674,38 @@ \subsection{Approximate equations}
 the dependence on the compositional fields used in previous sections, though
 this dependence can easily be added back into the equations stated below. A
 detailed discussion of the approximations introduced below can also be found in \cite{STO01} and
-\cite{KLKLZTTK10}.
+\cite{KLKLZTTK10}; a theoretical and practical comparison of many of
+these formulations using \aspect{} can be found in
+\cite{gassmoller2020formulations}.
+
+\note{Historically, the mantle convection community has typically used
+  one or another of these simplified models for computer simulations
```
Comment (bobmyhill) [inline doc/manual/manual.tex:1682]:
```suggestion
  one or another of these simplified formulations for computer simulations
```

File: doc/manual/manual.tex [hunk 6]
```diff
@@ -1674,7 +1674,38 @@ \subsection{Approximate equations}
...
+  \eqref{eq:stokes-1}--\eqref{eq:temperature}, but \textit{because
+    simulation technology did not allow for anything else at the
+    time}.
+
+  This has changed today, and \aspect{} implements more realistic
+  models as discussed in this section and in
+  Section~\ref{sec:choosing-a-formulation}. As a consequence, you
+  should evaluate which formulation and approximation is
+  appropriate for what you want to do -- just because someone else in
+  the past used a simplified model does not mean that you should do
```
Comment (bobmyhill) [inline doc/manual/manual.tex:1703]:
```suggestion
  the past used a simplified formulation does not mean that you should do
```

File: doc/manual/manual.tex [hunk 7]
```diff
@@ -1674,7 +1674,38 @@ \subsection{Approximate equations}
...
+  rely on these approximations because they describe the physical
+  situation better than equations
+  \eqref{eq:stokes-1}--\eqref{eq:temperature}, but \textit{because
+    simulation technology did not allow for anything else at the
+    time}.
+
+  This has changed today, and \aspect{} implements more realistic
+  models as discussed in this section and in
+  Section~\ref{sec:choosing-a-formulation}. As a consequence, you
+  should evaluate which formulation and approximation is
```
Comment (bobmyhill) [inline doc/manual/manual.tex:1701]:
```suggestion
  should evaluate which formulation is
```

File: doc/manual/manual.tex (hunk 3 above)
Comment (anne-glerum) [reply doc/manual/manual.tex:1702]: Somehow this formulation feels less harsh.

**Review by bobmyhill — COMMENTED** [review]
I've suggested changing `model` -> `formulation`, just to limit the number of ways that `model` is used in the manual. 

Otherwise this is good for me!

**Comment by bangerth** [comment]
Thanks, @anne-glerum  and @bobmyhill -- all applied as suggested!

**Comment by bobmyhill** [comment]
Great, thanks @bangerth!

**Pull request merged**